### ISSUE TITLE
Add alt text field for sketches when they are created for problem descriptions

### ISSUE THREAD

**Issue by NSoiffer** [benetech/MathShare#534]
Problem set creators can add a sketch to the problem description. This sketch is displayed on the problem set page. For accessibility, there should be a short description of what the sketch is.  This can only be generated by the person writing the problem so we should add an "Short textual description" below(?) the sketchpad so problem creators can create the description.

Implementation: this should be added to the problem and referenced in buildProblemImage().

**Comment by NSoiffer** [comment]
Additionally, when a student creates a sketch, they should be able to add some alt text; this should then be surfaced in the "steps" section, or fall back to a default ("sketch") alt if not present. This is useful if a student is sharing their work with someone with a visual disability.
